src/frontend/src/flows/register/finish.ts

```diff
@@ -39,6 +40,9 @@ export const displayUserNumberTemplate = ({
               elem.classList.add("is-copied");
             });
           } catch (e: unknown) {
+            toast.error(
+              "Unable to copy Identity Anchor, check the console for details"
```

Comment:
```suggestion
              "Unable to copy Identity Anchor"
```

I think anyone with some dev knowledge will know to check the console, but for everyone else this will just be confusing